controls/pcidss_4_ocp4.yml

```diff
@@ -1244,7 +1244,22 @@ controls:
         - The encryption strength is appropriate for the encryption methodology in use.
       levels:
         - base
-      status: pending
+      status: partial
+      notes: |-
+        OpenShift provides mechanisms to securely transmit PAN over open public networks, but
+        the application is still responsible for leveraging and implementing strong
+        cryptography when transmitting PAN.
+      rules:
+        - file_permissions_openshift_pki_cert_files
+        - tls_version_check_apiserver
+        - tls_version_check_masters_workers
+        - tls_version_check_router
+        - etcd_check_cipher_suite
+        - api_server_tls_security_profile
+        - ingress_controller_certificate
+        - ingress_controller_tls_security_profile
+        - kubelet_configure_tls_min_version
```

Comment:
These 4 rules are new additions.

Reply:
Do we also need to include encryption between various components? https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/api-server/api_server_etcd_ca/rule.yml

Reply:
@yuumasato Does the rule api_server_tls_cipher_suites related? Thanks.

Reply:
@xiaojiey Thank you, I added the rule, we need both `api_server_tls_security_profile` and `api_server_tls_cipher_suite`.